js/common.js

```diff
@@ -92,8 +92,10 @@ function addBuild(buildRevision) {
         var sources = yield (search({
             "limit": 10000,
             "format": "list",
-            "groupby": ["build.revision", "build.created_timestamp"],
```

Comment:
There are a lot of timestamps, grouping by them will give you slowness.  We will pick the latest coverage instead.